src/components/ha-slider.js

```diff
@@ -79,6 +79,10 @@ class HaSlider extends PaperSliderClass {
     return subTemplate;
   }
 
+  _setImmediateValue(newImmediateValue) {
```

Comment:
`<ha-slider>` is used in various contexts. Is this desired in all of them? 

CC @bramkragten

Reply:
I looked through all the other places, and I think it should be ok. We would have had the same issue with unrounded floats being displayed in the UI if it was an issue elsewhere.